learn/guides/distinctive-language-features/network-interaction.md

```diff
@@ -213,7 +213,7 @@ A single listener can have multiple services attached to it, each with different
 
 ## Resource path parameters
 
-The resource paths can also be parameterized such that instead of having fixed, static paths, they can be dynamically assigned during the service invocation.  
+The resource paths can also be parameterized such that instead of having fixed, static paths, they can be dynamically assigned during the resource access.  
```

Comment:
```suggestion
The resource paths can also be parameterized such that instead of having fixed, static paths, they can be assigned dynamically during the resource access.  
```